frontend/server/controllers/IdentityController.php

```diff
@@ -213,7 +213,7 @@ public static function apiUpdate(Request $r) {
         global $experiments;
         $experiments->ensureEnabled(Experiments::IDENTITIES);
         self::validateUpdateRequest($r);
-        $original_identity = self::resolveIdentity($r['username']);
+        $originalIdentity = self::resolveIdentity($r['originalIdentityUsername']);
```

Comment:
```suggestion
        $originalIdentity = self::resolveIdentity($r['original_username']);
```

por consistencia con cómo pasamos los argumentos a través del API.

Reply:
listo